packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -371,14 +371,16 @@ export class StripeHelper extends StripeHelperBase {
     uid: string,
     email: string,
     displayName: string,
-    idempotencyKey: string
+    idempotencyKey: string,
+    ipAddress?: string
   ): Promise<Stripe.Customer> {
     const stripeCustomer = await this.stripe.customers.create(
       {
         email,
         name: displayName,
         description: uid,
         metadata: { userid: uid },
+        ...{ tax: { ip_address: ipAddress } },
```

Comment:
Also, is it okay for this `tax` object to have an `undefined` `ipAddress` in the case that none is passed into the method?

It looks like it's an accepted type for Stripe's customer `create` method, but I'm curious what the impact is downstream when Stripe attempts to use this field to determine the tax to apply. From [the docs](https://stripe.com/docs/tax/subscriptions#create-a-customer), it looks like you'd probably get `'unrecognized_location'` for `automatic_tax` in the expanded `tax` field returned on the customer when requested.

Reply:
If Stripe updates this, then the type will change and we'll be flagged on it when building.